GianGue_solution/solution_area_of_a_room.py

```diff
@@ -0,0 +1,21 @@
+##resolution problem m1-001 
+#This program has been created to let people calucate the area of a room, given lenght and width.
+print('For calulate the area of a rectangular room please enter the following data: ')
+w=input('Enter the winth of the room in meters: ')
+l=input('Enter the lenght of the room in meters: ')
+w=float(w)
+l=float(l)
+print('The area of the room is:', w*l, 'm**2')
+
+#In case of a room square, please insert this data
+s=input('For a square please insert only the lengh of a side: ')
+s=float(s)
+print('The area of the room is:', s*s, 'm**2')
+
+#In case of a triangular room, please insert this data 
+print('For a triangular room, please insert this data: ')
```

Comment:
Wow, you did a great job, even more from the original requirement.

Be careful with this, don't try to do too much, unless you are already confident with your skills